Source/com/Communicator.cpp

```diff
@@ -452,11 +528,16 @@ PUSH_WARNING(DISABLE_WARNING_THIS_IN_MEMBER_INITIALIZER_LIST)
         _connectionMap.Destroy();
     }
 
-    void Communicator::Destroy(const uint32_t id) {
+    void Communicator::Destroy(const uint32_t id)
+    {
         // This is a forceull call, blocking, to kill that specific connection
         g_destructor.ForceDestruct(id);
     }
 
+    const std::vector<string> Process::DynamicLoaderPaths() const
```

Comment:
lets return a const std::vector<string>& (reference) since we do not need a copy here, we just need access to the orginal..

Reply:
done